src/routes/spent/index.ts

```diff
@@ -0,0 +1,10 @@
+import ExpenseController from '../spent/controller/ExpenseController'
+import {Router} from 'express'
+
+const spentRouter = Router()
+
+const expense = new ExpenseController()
+
+spentRouter.post('/create', expense.store)
```

Comment:
Como essa rota já pertence ao `/expenses`, seguindo o padrão REST, não precisaria do create

```suggestion
expensesRouter.post('/', expense.store)
```